contracts/CBKLandSale.sol

```diff
@@ -358,25 +430,53 @@ contract CBKLandSale is Initializable, AccessControlUpgradeable {
         _enabled = allowed;
     }
 
-    function reserveChunks(uint256[] calldata chunkIds, bool reserve) external isAdmin {
+    function setReservedSaleAllowed(bool allowed) external isAdmin {
+        _reservedEnabled = allowed;
+    }
+
+    function reserveChunks(uint256[] calldata chunkIds, address reserveFor, bool reserve, bool forced) external isAdmin {
```

Comment:
a small note on the reserve parameter, it could be a bit confusing. But since this is an admin only function it's not a big deal. Ideally the function or param name would indicate the two directions this function can set things.

Reply:
will rename to setChunksReservationInfo and rework a bit